client/src/colorPicker/colorPicker.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import { SketchPicker } from "react-color";
+
+interface Props {}
+// http://casesandberg.github.io/react-color/

```

Comment:
Does this have to be here?

Reply:
Let's leave it there for now for reference

Reply:
Agreed - it'll be helpful as a documentation source